### PR TITLE
DIV-6016 Added ClarificationDigital field

### PR DESCRIPTION
**Pull request by alexwakeman**
[Results of 'How do you want submit your documents?' field added to CCD](https://tools.hmcts.net/jira/browse/DIV-6016)

Adds single field & mapping

Corresponding config: https://github.com/hmcts/div-ccd-definitions/pull/273

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/DnCaseData.java [hunk 1]
```diff
@@ -90,4 +90,7 @@
 
     @JsonProperty("DesertionAskedToResumeDNDetails")
     private String desertionAskedToResumeDNDetails;
+
+    @JsonProperty("ClarificationDigital")
```
Comment (tchow8) [inline src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/DnCaseData.java]: Can you put this in the DnRefusalCaseData model please, with a JsonIgnore NONNULL on it.
Unless we are merging config first, in which case you can leave as is.
If this is merged into master first without the config in Prod, it'll break Prod most likely unless you do the above change.

Comment (tchow8) [reply src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/DnCaseData.java]: And use the DivorceCaseToDnClarificationMapper if this field will only be there for Dn Clarification?

**Comment by qzhou-hmcts** [comment]
Do you have config changes for this?

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/DnCaseData.java (hunk 1 above)
Comment (alexwakeman) [reply src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/DnCaseData.java]: Hi @tchow8 - I've changed it based on your feedback - how is it now?

**Comment by alexwakeman** [comment]
Config is updated in branch: https://github.com/hmcts/div-ccd-definitions/pull/273

**Pull request merged**